client/cody/src/main.ts

```diff
@@ -258,6 +259,8 @@ const register = async (
     )
 
     if (initialConfig.completions) {
+        const statusBar = createStatusBar()
```

Comment:
```suggestion
    const statusBar = createStatusBar()
    disposables.push(statusBar)
    if (initialConfig.completions) {
```

Shouldn't we display the logo even if completions is not enabled so that users can still toggle other features using the status bar?

Reply:
YES sorry that was a total oversight I added it _only_ for completions first. :D